packages/fxa-settings/README.md

```diff
@@ -1,75 +1,182 @@
 # Firefox Accounts Settings
 
+This documentation is up to date as of 2020-06-08.
+
 ## Development
 
-- `npm run start|stop|restart` to start, stop, and restart the server as a PM2 process
-- `npm run build` to create a production build
+- `yarn start|stop|restart` to start, stop, and restart the server as a PM2 process
+- `yarn build` to create a production build
+- `yarn test` to run unit tests
+- `yarn storybook` to open Storybook
 
 ### External imports
 
-You can import React components into this project. This is currently restricted to `fxa-react`:
+You can import React components into this project from other React projects. This is currently restricted to `fxa-react`:
 
 ```javascript
 // e.g. assuming the component HelloWorld exists
 import HelloWorld from 'fxa-react/components/HelloWorld';
 ```
 
+Components from other React packages can be moved into `fxa-react` to be shared across multiple packages.
+
 ### Styling components
 
 #### Tailwind
 
-The package uses [Tailwind CSS](https://tailwindcss.com/) for the bulk of its styles. If you're not familiar you're encouraged to [learn more](https://tailwindcss.com/docs/utility-first), but the idea is simple: use predefined classes on elements to layer on individual styles. **You can accomplish most of your styling needs with the classes provided.**
+At the time of writing, the `fxa-settings`, `fxa-react`, and `fxa-admin-panel` packages are setup to share a [Tailwind CSS](https://tailwindcss.com/) configuration file found in the `fxa-react` package. If you're not familiar with Tailwind, look through [their documentation](https://tailwindcss.com/docs) to get an idea of what [utility-first](https://tailwindcss.com/docs/utility-first) (Atomic CSS) is and what you can expect while using it. The general idea is simple: use single-purpose classes on elements to layer styles until the design is achieved. **You can accomplish almost all of your styling needs with classes provided by Tailwind's default configuration or through adding them in the configuration file.**
+
+Each package has its own `tailwind.css` file. This file, any S/CSS files this file imports, and the Tailwind configuration file are compiled to produce `tailwind.out.css`. Rebuilding this file should happen automatically on any change. If you're not sure what specific class name corresponds with the style you need or if you need to check that a specific style exists in our utility classes, search through `tailwind.out.css` for these styles as needed.
```

Comment:
> search through `tailwind.out.css` for these styles as needed

This might be an edge case, but there have been times where I had most recently run a command that generates a production build of `tailwind.out.css`, and then went to do some dev work and was so confused why my classes weren't in that file. The auto-rebuilding that Danny added might solve this, but I figured I'd note it here.

Reply:
Ah interesting, that's probably an edge case now because it does watch and auto rebuild, but if you run a "build" command and don't go edit one of the files, you're probably stuck with the production build file so I'll make note.